pkgs/tools/admin/eksctl/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
```

Comment:
```suggestion
{ lib, buildGoPackage, fetchFromGitHub }:
```
If you're only using `stdenv.lib` you might as well just use `lib` directly here.